app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
```

Comment:
Ensure consistent use of double quotes throughout your code for better readability and adherence to style guidelines as per the checklist.